en/docs/how-tos/passing-enduser-attributes-to-the-backend-using-jwt.md

```diff
@@ -175,6 +175,69 @@ Outer `[jwtGeneratorConfig]` configuration contains the properties and the custo
 </tbody>
 </table>
 
+#### Adding Claims Retrieved from a remote endpoint
+
+If the user needs to retrieve user claims from a remote endpoint, the microgateway can be configured to support that 
+as well. The microgateway has an inbuilt implementation to get user claims from 
+`<WSO2 Key Manager URL>/keymanager-operations/user-info/claims/generate` endpoint. If you are willing to use the same 
+implementation, you can add the following configuration to the `micro-gw.conf` file.
+
+```toml
+[jwtGeneratorConfig.claimRetrieval]
+      retrieverImpl="org.wso2.micro.gateway.jwt.generator.MGWClaimRetrieverImpl"
+      [jwtGeneratorConfig.claimRetrieval.configuration]
+          # The User can provide any key-value pair here. These are the properties used by default.
+          # serverUrl = <Key Manager URL>
```

Comment:
Shall we remove comment symbols and provide example values?

Reply:
updated via https://github.com/wso2/docs-mg/pull/43/commits/9480fc53f208b95446e6f3e220351d4b5229fa1c